src/hooks/useSortAndSearch.jsx

```diff
@@ -0,0 +1,53 @@
+import { useMemo } from 'react';
+import { SortSex } from '../types/SortSex';
+import { Sort } from '../types/Sort';
+
+export const useSortAndSearch = (people, sex, centuries, query, sort) => {
+  const modifiedPeople = useMemo(() => {
+    return (people
+      .filter(person => {
+        switch (sex) {
+          case SortSex.female:
+            return person.sex === 'f';
+          case SortSex.male:
+            return person.sex === 'm';
+          default:
+          case SortSex.all:
+            return person;
```

Comment:
There is a redundant case label for 'SortSex.all', as the default case is doing the same thing. Remove the 'SortSex.all' case.